index.js

```diff
@@ -109,6 +109,18 @@ const allEditors = () => [
 			'/Applications/Emacs.app/Contents/MacOS/Emacs'
 		],
 		keywords: []
+	},
+	{
+		id: 'androidstudio',
+		name: 'Android Studio',
+		binary: 'studio',
+		isTerminalEditor: false,
+		paths: [
+			'/Applications/Android Studio.app/Contents/MacOS/studio'
```

Comment:
This needs to include the paths on Windows and Linux too.

Reply:
Other entries could also have that, I was not sure about it, added for Android Studio.